packages/stitch/tests/blah.test.ts

```diff
@@ -0,0 +1,327 @@
+  }
+  if (!isOutputType(type)) {
+    throw new Error(`${name} is not an output type`);
+  }
+
+  return type;
+};
+
+describe('Blah', () => {
+  let aggSchema = makeExecutableSchema({
+    typeDefs: /* GraphQL */ `
+      type Query {
+        ccp: CcpQuery
+      }
+
+      type CcpQuery {
+        ccpEntitlement(id: ID!): CcpEntitlement
+        ccpProduct(id: ID!): CcpProduct
+      }
+
+      type CcpEntitlement {
+        id: ID!
+        offeringId: ID
+      }
+
+      type CcpProduct {
+        id: ID!
+        name: String
+      }
+    `,
+  });
+
+  let bbfSchema = makeExecutableSchema({
+    typeDefs: /* GraphQL */ `
+      type Query {
+        ccpOffering(id: ID!): CcpOffering
+      }
+
+      type CcpOffering {
+        id: ID!
+        productId: ID
+      }
+    `,
+  });
+
+  let stitchedSchema: GraphQLSchema;
+
+  let aggResolvers;
+
+  let bbfResolvers: any;
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  beforeEach(() => {
+    aggResolvers = {
+      CcpQuery: {
+        ccpEntitlement: async (_source: any, args: { id: string }) => {
+          return {
+            id: args.id,
+            offeringId: args.id,
+          };
+        },
+        ccpProduct: async (_source: any, args: { id: string }) => {
+          return {
+            id: args.id,
+            name: args.id,
+          };
+        },
+      },
+    };
+
+    bbfResolvers = {
+      Query: {
+        ccpOffering: async (_source: any, args: { id: string }) => {
+          return {
+            id: args.id,
+            productId: args.id,
+          };
+        },
+      },
+    };
+
+    aggSchema = addMocksToSchema({
+      schema: aggSchema,
+      resolvers: aggResolvers,
+    });
+    bbfSchema = addMocksToSchema({
+      schema: bbfSchema,
+      resolvers: bbfResolvers,
+    });
+
+    stitchedSchema = stitchSchemas({
+      subschemas: [{ schema: aggSchema }, { schema: bbfSchema }],
+      resolverValidationOptions: { requireResolversForResolveType: 'ignore' },
+      typeDefs: /* GraphQL */ `
+        extend type CcpEntitlement {
+          offering: CcpOffering
+        }
+
+        extend type CcpOffering {
+          product: CcpProduct
+        }
+      `,
+      resolvers: {
+        CcpEntitlement: {
+          offering: {
+            selectionSet: forwardArgsToSelectionSet('{ offeringId }'),
+            resolve: (parent, _args, context, info) => {
+              return delegateToSchema({
+                schema: bbfSchema,
+                operation: OperationTypeNode.QUERY,
+                fieldName: 'ccpOffering',
+                args: { id: parent.offeringId },
+                context,
+                info,
+              });
+            },
+          },
+        },
+        CcpOffering: {
+          product: {
+            selectionSet: forwardArgsToSelectionSet('{ productId }'),
+            resolve: async (parent, _args, context, info) => {
+              const fieldNode = getFieldNode(info);
```

Comment:
I think a fieldNode is an instance of a field in a selection set.
e.g. the following query has two fieldNodes `entitlement1` and `entitlement2` for the field `entitlement`:
```graphql
query {
  entitlement1: entitlement(id: "abc") { id }
  entitlement1: entitlement(id: "123") { id }
}
```

So probably this will break if that happens.